pkg/template/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": 0,
+    "name": "PACKAGE_NAME",
```

Comment:
Most packages shouldn't specify their name in their manifest. It should be implied from the directory name.